SECURITY.md

```diff
@@ -0,0 +1,35 @@
+# Security Policy
+
+## Supported Versions
+
+Redis is generally backwards compatible with very few exceptions, so we
+recommend users to always use the latest version to experience stability,
+performance and security.
+
+We generally backport security issues to a single previous major version,
+unless this is not possible or feasible with a reasonable effort.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 7.2.x   | :white_check_mark: |
+| 7.0.x   | :white_check_mark: |
+| 6.2.x   | :white_check_mark: |
+| < 6.2   | :x:                |
```

Comment:
shouldn't this table refer to RediSearch versions?

Reply:
Changes made!